File: GAPInsurance.API/GAPInsurance.API/Controllers/Requests/PolicyCreationRequest.cs
namespace GAPInsurance.API.Controllers.Requests {
  public class PolicyCreationRequest {
    public string Name { get; set; }
    public string Description { get; set; }
    public float PremiumPrice { get; set; }
    public string CoverageStartDate { get; set; }
    public int CoverageLength { get; set; }
    public int RiskLevelId { get; set; }
    public int EarthquakeCoverage { get; set; }
    public int FireCoverage { get; set; }
    public int TheftCoverage { get; set; }
    public int LossCoverage { get; set; }
  }
}

File: GAPInsurance.Client/src/models/policycreation.request.ts
export class PolicyCreationRequest {
  constructor(
    public name: string,
    public description: string,
    public premiumPrice: number,
    public coverageStartDate: string,
    public coverageLength: number,
    public riskLevelId: number,
    public earthquakeCoverage: number,
    public fireCoverage: number,
    public theftCoverage: number,
    public lossCoverage: number
  ) { }
}
